src/main/java/Storage.java

```diff
@@ -0,0 +1,135 @@
+import java.io.*;
+import java.io.BufferedReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    protected static ArrayList<Task> taskList = new ArrayList<Task>();
+    protected static String file = "todo.txt";
+
+    public Storage(String file) {
```

Comment:
You should consider removing this parameter, since it doesn't seem to do anything.